test/GameControllerTest.java

```diff
@@ -0,0 +1,101 @@
+import controllers.GameController;
+import controllers.routes;
+import org.junit.Before;
+import org.junit.Test;
+import play.Application;
+import play.inject.guice.GuiceApplicationBuilder;
+import play.mvc.Result;
+import play.test.WithApplication;
+import ttt.game.GameEngine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static play.mvc.Http.Status.OK;
+import static play.test.Helpers.contentAsString;
+import static play.test.Helpers.fakeRequest;
+import static play.test.Helpers.route;
+
+public class GameControllerTest extends WithApplication {
+
+    private GameController controller;
+
+    @Override
+    protected Application provideApplication() {
+        return new GuiceApplicationBuilder()
+                .configure("play.http.router", "router.Routes")
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        this.controller = new GameController();
+    }
+
+    @Test
+    public void gamePage() {
+        Result result = route(fakeRequest("GET", "/"));
+        assertEquals(OK, result.status());
+    }
+
+    @Test
+    public void gamePageShowsBoard() {
+        Result result = route(routes.GameController.showBoard());
+        assertTrue(contentAsString(result).contains("1"));
```

Comment:
Is there a better to assert that the game page shows the board other than asserting the page has a value "1"?